crates/orderbook/src/orderbook.rs

```diff
@@ -261,6 +261,14 @@ impl Orderbook {
         }
     }
 
+    /// Adds an order to the order book, placing it in the database.
```

Comment:
> Adds an order to the order book

Not sure what that means.
```suggestion
    /// Validates and stores an order in the database.
```

Reply:
Good suggestion. It's always hard to do "non-obvious" docs for methods that explain what they do at an high level through the name